src/dialects/sqlite3/schema/tablecompiler.js

```diff
@@ -132,7 +132,7 @@ TableCompiler_SQLite3.prototype.renameColumn = function(from, to) {
 
 TableCompiler_SQLite3.prototype.dropColumn = function() {
   const compiler = this;
-  const columns = Object.values(arguments);
+  const columns = values(arguments);
```

Comment:
Apparently Object.values is one thing that Babel has no idea how to transpile, and polyfilling just for this one case feels like an overkill.